change/@fluentui-react-trigger-878d2c89-74e6-475d-bd81-c43652fb8f57.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
```

```suggestion
  "type": "patch",
```

you should revert the manual bump you did and just change the change type in the change file. Beachball will automatically bump it to the next available non-prerelease version

This is why I'm not a fan of magic tools like beachball :D